.github/workflows/auto-label-issues.yml

```
@@ -29,6 +39,7 @@ jobs:
             echo "User is within the organization, labeling internal"
             gh issue edit "$NUMBER" --add-label "internal-issue"
           fi
+    timeout-minutes: 15
```

actionlint (1.7.12)

[error] key "timeout-minutes" is duplicated in "check-external" job. previously defined at line:16,col:5

(syntax-check)